decidim-admin/spec/features/manage_participatory_processes_spec.rb

```diff
@@ -0,0 +1,71 @@
+# frozen_string_literal: true
+
+require "spec_helper"
+
+describe "Manage participatory processes", type: :feature do
+  let(:admin) { create(:user, :admin) }
+  let(:participatory_process) { create(:process) }
+  let!(:participatory_process2) { create(:process) }
```

Comment:
If it's only used in one example I'd move it to that context to avoid creating it each time.